pygfunction/examples/custom_borehole.py

```diff
@@ -124,7 +129,25 @@ def main():
     R_b = gt.pipes.borehole_thermal_resistance(
         Coaxial, m_flow_borehole, fluid.cp)
 
+    # GLHEDT UBWT effective borehole resistance calculation
+    # Grout thermal resistance
+    R_grout = gt.pipes.conduction_thermal_resistance_circular_pipe(
+        r_out_out, borehole.r_b, k_g)
+    Ra = R_conv_1 + R_cyl_1 + R_conv_2
+    Rb = R_conv_3 + R_cyl_2 + R_grout
+    R_12 = Ra
+    eta = H / (fluid.rho * fluid.cp * V_flow_borehole) * 1 / (2 * Rb) * \
+          np.sqrt(1 + 4 * Rb / R_12)  # eq. (3.69)
+    Rb_star_UBH = Rb * eta * (1 / np.tanh(eta))  # eq. (3.68)
+
+    Rb_ = Coaxial._Rd[1, 1]  # Local borehole thermal resistance
+    assert Rb_ == Rb
+    assert R_ff == Ra
```

Comment:
Yes, I see what you are saying. The assertions are true here.

Reply:
However, running the script comes up with 

```
Coaxial tube Borehole thermal resistance: 0.1789 m.K/W
Effective borehole resistance m.K/W(GLHEDT): 0.126910
```

Therefore, equation 3.69 and 3.68 are computing a different effective borehole thermal resistance. Why?

Reply:
Something strange here is that we should not find 'R_b^star < R_b (R_b = R_fg)'. Is `R_fg` the same in both tools?

Reply:
This line `Rb_star_UBH = Rb * eta * (1 / np.tanh(eta))  # eq. (3.68)` hardly does anything (< 1.0e-06 difference) over a wide range of flow rates.

Reply:
> Something strange here is that we should not find `R_b^star < R_b (R_b = R_fg)`. Is `R_fg` the same in both tools?

`R_fg` is the local borehole thermal resistance right? The answer is yes, they are identical. The line `assert Rb_ == Rb` has to be updated to `assert abs(Rb_ - Rb) < 1.0e-6`. They are identical.

Reply:
> This line `Rb_star_UBH = Rb * eta * (1 / np.tanh(eta)) # eq. (3.68)` hardly does anything (< 1.0e-06 difference) over a wide range of flow rates.

I believe it could be possible that there is something wrong here. This is practically local borehole resistance. Hellstrom's thesis describes the delta circuit for annular dicts on pages 79-80. 

`pygfunction` is finding a different value for effective borehole resistance (`Rb*`) than local borehole resistanace, the GLHEDT method is not. There is going to be short circuiting in a concentric annulus. I'd think there would be more short circuiting in a concentric tube heat exchanger than a U-tube arrangement where the tubes are separated by grout.